tests/back/Pim/Structure/Specification/Bundle/EventSubscriber/SaveFamilyVariantOnFamilyUpdateSubscriberSpec.php

```diff
@@ -245,4 +230,31 @@ function it_does_not_bulk_save_if_on_non_unitary_save_and_POST_SAVE_ALL_event(
 
         $this->onBulkSave($event);
     }
+
+    function it_does_not_force_launch_compute_family_variant_job_when_attributes_did_not_change(
+        ValidatorInterface $validator,
+        BulkSaverInterface $bulkFamilyVariantSaver,
+        GenericEvent $event,
+        ConstraintViolationList $constraintViolationList,
```

Comment:
```suggestion
        ConstraintViolationList $constraintViolationList
```
php 7.4 :slightly_smiling_face: